.github/workflows/test.yml

```diff
@@ -90,5 +50,5 @@ jobs:
 
       - name: Test
         run: |
-          pytest
+          PY_IGNORE_IMPORTMISMATCH=1 pytest
```

Comment:
Why is it needed to set this env var?

Reply:
I'm not entirely sure, I think it may be due to distutils? Without, it fails like this:

```
_______________ ERROR collecting quantities/tests/test_units.py ________________
import file mismatch:
imported module 'quantities.tests.test_units' has this __file__ attribute:
  /opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/quantities/tests/test_units.py
which is not the same as the test file we want to collect:
  /home/runner/work/python-quantities/python-quantities/quantities/tests/test_units.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
```

https://github.com/hugovk/python-quantities/runs/3709626106?check_suite_focus=true

See also: https://github.com/pytest-dev/pytest/issues/2042